reinforcement_learning/include/ranking_response.h

```diff
@@ -129,31 +132,77 @@ namespace reinforcement_learning {
      */
     ranking_response& operator =(const ranking_response&) = delete;
   private:
-    ranking_response_impl* _pimpl; //!The actual implementation details are forwarded to this object (PIMPL pattern)
+    std::string _event_id;
+    size_t _chosen_action_id;
+    std::string _model_id;
+    using coll_type = std::vector<action_prob>;
```

Comment:
[nit] I think we have "_t" everywhere else for type naming

Reply:
Done.

---
In reply to: [228962208](https://github.com/VowpalWabbit/vowpal_wabbit/pull/1658#discussion_r228962208) [](ancestors = 228962208)